gtk-vnc.rb

```diff
@@ -0,0 +1,46 @@
+class GtkVnc < Formula
+  desc "VNC viewer widget for GTK."
+  homepage "https://wiki.gnome.org/Projects/gtk-vnc"
+  url "https://download.gnome.org/sources/gtk-vnc/0.5/gtk-vnc-0.5.4.tar.xz"
+  sha256 "488aa97a76ce6868160699cd45d4a0ee0fe6f0ad4631737c6ddd84450f6c9ce7"
+
+  depends_on "gobject-introspection" => :optional
+  depends_on "gettext" => :build
+  depends_on "gdk-pixbuf"
+  depends_on "gnutls"
+  depends_on "gtk+3"
+  depends_on "intltool" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+  depends_on "pulseaudio" => :optional
+  depends_on "vala" => :optional
+
+  def install
+    args = ["--prefix=#{prefix}",
+            "--with-gtk=3.0",
+            "--with-examples",
+            "--with-python"]
+
+    args << "--enable-introspection" if build.with? "gobject-introspection"
+    args << "--enable-pulseaudio" if build.with? "pulseaudio"
+    if build.with? "vala"
+      args << "--enable-vala"
+    else
+      args << "--disable-vala"
+    end
+
+    # fix "The deprecated ucontext routines require _XOPEN_SOURCE to be defined"
+    ENV.append "CPPFLAGS", "-D_XOPEN_SOURCE=600"
+    # for MAP_ANON
+    ENV.append "CPPFLAGS", "-D_DARWIN_C_SOURCE"
+
+    system "./configure", *args
+
+    system "make"
+    system "make", "install"
+  end
+
+  test do
+    File.exist? "#{bin}/gvncviewer"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!